assets/base.css

```diff
@@ -3127,6 +3127,7 @@ details-disclosure > details {
 .animate--fixed > img:not(.zoom):not(.deferred-media__poster-button),
 .animate--fixed > svg:not(.zoom):not(.deferred-media__poster-button) {
   position: fixed;
+  height: 100vh;
```

Comment:
Should we pair this with a `width: auto` to safeguard against stretching if future styles interfere here? I'm always overly cautious about images when it comes to that. 😅

Reply:
yeah I agree, I was thinking of that but also not seeing an impact on the images I was testing. Probably safer to add it 👍

Reply:
Actually messes with things 😅 and we already have declarations that set the width and max-width to `100%` while the height is set to `100% too. 

https://user-images.githubusercontent.com/28613840/236268356-8d5f4922-ee8d-4b9e-93ad-6c058cd3a51f.mp4

I could also set the height on the image wrapper instead 🤔

Reply:
Oh well. We've got another `width: 100%` rule on the wrapper too, so I'm not sure that's much better. I think what we have here seems to work ok.